styles.css

```diff
@@ -0,0 +1,112 @@
+* {
+margin: 0;
+padding: 0;
+box-sizing: border-box;
+}
+
+.app {
+    width: 100vw;
+    min-height: 100vh;
+    background-image: url("images/background.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+
+}
+
+
+.nav-bar-list {
+    display: flex;
+    gap: 32px;
+    align-items: center;
+}
+
+.nav-bar-list-right {
+    display: flex;
+    gap: 12px;
+    align-items: center;
+}
+
+
+.nav-link {
+    list-style: none;
+}
+
+
+.nav-link-text {
+    font-family: 'inter', sans-serif;
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 20px;
+    color: #18191f;
+    text-decoration: none;
+    
+}
+
+.nav-bar {
+    max-width: 1440px;
+    height: 72px;
+    display: flex;
+    align-items: center;
+    padding: 0 20px;
+    justify-content: space-between;
+    margin: auto;
+}
+
+
+.hero-section {
+    display: flex;
+    flex-direction: column;
+    max-width: 1440px;
+    margin: 100px auto auto;
+    padding: 0 20px;
+    
+}
+
+.hero-title {
+    font-family: 'Inter', sans-serif;
+    font-weight: 800;
+    font-size: 72px;
+    line-height: 98px;
+    color: #18191F;
+    margin-top: 70px;
+}
+
+
+.hero-content-wrapper {
+   display: flex;
+   justify-content: space-between;
+}
+
+
+.hero-content {
+    display: flex;
+    flex-direction: column;
+    gap: 32px;
+}
+
+.hero-button {
+    width: 154px;
+    height: 48px;
+    background: #8C30F5;
+    border-radius: 6px;
+    color: #FFFFFF;
+    border: 0;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    cursor: pointer;
+}
+
+.hero-company-logos {
+    height: 110px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: auto;
```

Comment:
- [ ] Tasarımda üstte ve altta boşluklar var onun için marginde böyle yapabilirsin  `margin: 50px auto 50px auto;`
- [ ] Bir de şu box shadow tasarımdakine daha yakın gibi ^^ `box-shadow: rgba(0, 0, 0, 0.16) 0px 1px 4px;`